lib/modules/@apostrophecms/storybook/template/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Apostrophe Technologies, Inc.",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build-storybook": "build-storybook -s ./.storybook/public -o .out"
+    "build-storybook": "build-storybook -s .storybook/public -o .out"
```

Comment:
Is this still being used? If the `deploy-storybook` isn't building anymore I don't think it would be.